packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/header-hamburger.twig

```diff
@@ -10,13 +10,14 @@
 
   <a class="ma__header__hamburger__skip-nav" href="#main-content">skip to main content</a>
 
-  <nav class="ma__header__hamburger__nav" aria-label="main navigation" id="main-navigation" role="navigation">
+  <nav class="ma__header__hamburger__nav" id="main-navigation" role="navigation">
+    <span id="header-nav-label" class="ma__visually-hidden">Main navigation</span>

```

Comment:
You don't think it needs a label to group all navigation units as a whole?  I was questioning the necessity of it, but didn't come to a conclusion.

Reply:
I think this should be an aria-label instead of a piece of standalone text. We can put this as an aria-label in the nav tag, see the APG example https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation/